api/build.rb

```diff
@@ -6,12 +6,12 @@
 
 def merge_yaml()
   Workbench::Swagger::merge_yaml('src/main/resources/workbench.yaml',
-    ['src/main/resources/client_api.yaml'], 'src/main/resources/merged.yaml')
+    ['src/main/resources/client_api.yaml', 'src/main/resources/cb_review_api.yaml', 'src/main/resources/cb_search_api.yaml'], 'src/main/resources/merged.yaml')
 end
 
 Common.register_command({
   :invocation => "merge-yaml",
-  :description => "Merges client_api.yaml and workbench.yaml together to produce a single YAML spec.",
+  :description => "Merges client_api.yaml, cb_review_api.yaml, cb_search_api.yaml and workbench.yaml together to produce a single YAML spec.",
```

Comment:
We can probably just say "Merges API YAML files together"

Reply:
Done.